### ISSUE TITLE
<3

### ISSUE THREAD

**Issue by PvathlP** [byte2mov/gelion-bypass#2]
Contributers : Vathl (Vathl designed the general idea of the hook and proposed it to me.)
nigga really said **proposed** to me

**Comment by PvathlP** [comment]
i will beat yo fat ass

**Comment by PvathlP** [comment]
stolen code

**Comment by byte2mov** [comment]
Yes, like the credits said you designed the general hook, but file downloading and every else I did in the source was done by me, what did you do that I didn’t give you credits for ?

**Comment by PvathlP** [comment]
you made it better so idc <3

**Comment by byte2mov** [comment]
:)

**Comment by PvathlP** [comment]
btw you got a cracking server?
